spec/spec_helper.rb

```diff
@@ -17,10 +17,16 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
-require "rails-patterns"
+require "active_support/all"
+require "active_support/testing/time_helpers"
+
+require "rspec/mocks/standalone"
```

Comment:
@stevo That's the only way to use rspec-mocks (so `allow` or `double`) in `before(:all)` 😄
I can create an actual class instead.